go/consensus/api/submission.go

```diff
@@ -51,6 +53,8 @@ type SubmissionManager interface {
 	//
 	// It also automatically handles retries in case the nonce was incorrectly estimated.
 	SignAndSubmitTx(ctx context.Context, signer signature.Signer, tx *transaction.Transaction) error
+
+	// TODO: Add SubmitSignedTx
```

Comment:
Why would this be needed, if you have a signed tx and don't have the corresponding signers, you can just use the regular `SubmitTx` on the consensus backend?

Reply:
Sure, ok.